docs/code-examples/Cargo.toml

```diff
@@ -1,14 +1,116 @@
 [package]
 name = "code_examples"
-version = "0.7.0-alpha.0"
-edition = "2021"
-rust-version = "1.69"
-license = "MIT OR Apache-2.0"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
 publish = false
 
+
+[[bin]]
+name = 'annotation_context_connections'
```

Comment:
Oh man, I didn't even know single-quote strings were a thing in TOML... I'm actually very surprised that `taplo` let's you get away with this...

Can you switch it to double-quotes please? For consistency / idiomaticity.

Reply:
I spent too much time with Python / Black to even see the difference :) Will do.